data-raw/bucharest.R

```diff
@@ -51,24 +51,16 @@ get_osm_river <- function(river_name, bb, crs) {
   river_centerline <- osmdata_as_sf("waterway", "river", bb)
   river_centerline <- river_centerline$osm_multilines |>
     filter(.data$name == river_name) |>
+    st_filter(sf::st_as_sfc(bb), .predicate = sf::st_intersects) |>
```

Comment:
As also noticed in CRiSp, the Overpass API query for the river centerline now returns an additional branch of the Dambovita, not sure why, since it does not intersects the bounding box...